[2022-04-29 15:12:35,341] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: cdn_daily.import_cat.check_presence_of_cat2 manual__2022-04-29T15:12:33.087393+00:00 [queued]>
[2022-04-29 15:12:35,366] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: cdn_daily.import_cat.check_presence_of_cat2 manual__2022-04-29T15:12:33.087393+00:00 [queued]>
[2022-04-29 15:12:35,368] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-29 15:12:35,369] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-04-29 15:12:35,371] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-29 15:12:35,399] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): import_cat.check_presence_of_cat2> on 2022-04-29 15:12:33.087393+00:00
[2022-04-29 15:12:35,409] {standard_task_runner.py:52} INFO - Started process 2772 to run task
[2022-04-29 15:12:35,417] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cdn_daily', 'import_cat.check_presence_of_cat2', 'manual__2022-04-29T15:12:33.087393+00:00', '--job-id', '1599', '--raw', '--subdir', 'DAGS_FOLDER/cdn_daily.py', '--cfg-path', '/tmp/tmpjppl8by3', '--error-file', '/tmp/tmp93mtqbon']
[2022-04-29 15:12:35,419] {standard_task_runner.py:77} INFO - Job 1599: Subtask import_cat.check_presence_of_cat2
[2022-04-29 15:12:35,557] {logging_mixin.py:109} INFO - Running <TaskInstance: cdn_daily.import_cat.check_presence_of_cat2 manual__2022-04-29T15:12:33.087393+00:00 [running]> on host d5ad096ffea5
[2022-04-29 15:12:35,685] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cdn_daily
AIRFLOW_CTX_TASK_ID=import_cat.check_presence_of_cat2
AIRFLOW_CTX_EXECUTION_DATE=2022-04-29T15:12:33.087393+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-29T15:12:33.087393+00:00
[2022-04-29 15:12:35,754] {python.py:175} INFO - Done. Returned value was: None
[2022-04-29 15:12:35,813] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=cdn_daily, task_id=import_cat.check_presence_of_cat2, execution_date=20220429T151233, start_date=20220429T151235, end_date=20220429T151235
[2022-04-29 15:12:35,878] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-29 15:12:35,996] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
